src/main/java/com/spring/boot/services/ProjectServiceImpl.java

```diff
@@ -38,8 +38,8 @@ public void updateProject(Project project) {
 	}
 
 	@Override
-	public Optional<Project> getProjectById(Long id) {
-		return projectRepo.findById(id);
+	public Project getProjectById(Long id) {
+		return projectRepo.findById(id).get();
```

Comment:
change name projectRepo to projectRepository..

Reply:
Okey akka